policyengine_us/variables/gov/states/nc/ncdhhs/scca/nc_scca_fpl_rate.py

```diff
@@ -0,0 +1,39 @@
+from policyengine_us.model_api import *
+
+
+class nc_scca_fpl_rate(Variable):
```

Comment:
we use the more formal fpg (guideline) abbreviation - change elsewhere too
```suggestion
class nc_scca_fpg_rate(Variable):
```